api/nodes/views.py

```diff
@@ -179,13 +241,105 @@ def perform_create(self, serializer):
-    is that a project is the top-level node, and components are children of the project. There is also a category field
-    that includes the option of project. The categorization essentially determines which icon is displayed by the
-    Node in the front-end UI and helps with search organization. Top-level Nodes may have a category other than
-    project, and children nodes may have a category of project.
+    is that a project is the top-level node, and components are children of the project. There is also a [category
+    field](/v2/#osf-node-categories) that includes the option of project. The categorization essentially determines
+    which icon is displayed by the node in the front-end UI and helps with search organization. Top-level nodes may have
+    a category other than project, and children nodes may have a category of project.
+
+    ###Permissions
+
+    Nodes that are made public will give read-only access to everyone. Private nodes require explicit read
+    permission. Write and admin access are the same for public and private nodes. Administrators on a parent node have
+    implicit read permissions for all child nodes.
+
+    ##Attributes
+
+    OSF Node entities have the "nodes" `type`.
+
+        name           type               description
+        ---------------------------------------------------------------------------------
+        title          string             title of project or component
+        description    string             description of the node
+        category       string             node category, must be one of the allowed values
+        date_created   iso8601 timestamp  timestamp that the node was created
+        date_modified  iso8601 timestamp  timestamp when the node was last updated
+        tags           array of strings   list of tags that describe the node
+        registration   boolean            has this project been registered?
+        collection     boolean            is this node a collection of other nodes?
+        dashboard      boolean            is this node visible on the user dashboard?
+        public         boolean            has this node been made publicly-visible?
+
+    ##Relationships
+
+    ###Children
+
+    List of nodes that are children of this node.  New child nodes may be added through this endpoint.
+
+    ###Contributors
+
+    List of users who are contributors to this node.  Contributors may have "read", "write", or "admin" permissions.  A
+    node must always have at least one "admin" contributor.  Contributors may be added via this endpoint.
+
+    ###Files
+
+    List of top-level folders (actually cloud-storage providers) associated with this node. This is the starting point
+    for accessing the actual files stored with this node.
+
+    ###Parent
+
+    If this node is a child node of another node, the parent's canonical endpoint will be available in the
+    `parent.links.self.href` key.
```

Comment:
If a node does not have a parent, the value at the `parent` key will be null. 

Also we should consider using JSON pointers for describing object paths: https://tools.ietf.org/html/rfc6901